src/components/shared/HeaderMessage.tsx

```diff
@@ -0,0 +1,13 @@
+interface props {
+  heading: string;
+  subheading: string;
+}
+
+export default function HeaderMessage({ heading, subheading }: props) {
+  return (
+    <div className="inline-flex md:flex lg:flex flex-col items-start gap-2">
+      <h2 className="text-neutral-800 text-center">{heading}</h2>
```

Comment:
```suggestion
      <h2 className="text-neutral-800">{heading}</h2>
```
No need to center the text here. I know the Figma component uses a center align, but that was unintentional.